src/styles/catalog.scss

```
@@ -0,0 +1,34 @@
+
+
+.catalog {
+  display: grid;
+  justify-content: center;
+  gap: $cardGap;
+  padding-bottom: 48px;
+
+  margin: 0 auto;
+
+  @media (min-width: 488px) {
+    max-width: calc(#{$cardWidth} * 2 + #{$cardGap});
+    grid-template-columns: repeat(2, $cardWidth);
+  }
+
+  @media (min-width: 768px) {
+    max-width: calc(#{$cardWidth} * 3 + #{$cardGap} * 2);
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (min-width: 1024px) {
+    max-width: calc(#{$cardWidth} * 4 + #{$cardGap} * 3);
+    grid-template-columns: repeat(4, $cardWidth);
```

you don't need `max-width`
```suggestion
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, $cardWidth);
  }

  @media (min-width: 768px) {
    grid-template-columns: repeat(3, $cardWidth);
  }

  @media (min-width: 1024px) {
    grid-template-columns: repeat(4, $cardWidth);
```